inc/toolbox.class.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+class PluginFieldsToolbox {
+
+   /**
+    * Get a clean system name from a label.
+    *
+    * @param string $label
+    *
+    * @return string
+    */
+   public function getSystemNameFromLabel($label) {
+
+      $name = strtolower($label);
+
+      // 1. remove trailing "s" (plural forms)
+      $name = preg_replace('/s*$/', '', $name);
```

Comment:
There are methods in core to manage plural/singular

Reply:
Right, thanks.